Cargo.toml

```diff
@@ -22,6 +22,7 @@ opt-level = 1
 
 [profile.release]
 lto = true
+strip = true
```

Comment:
This does not belong in this PR and i'm also not sure if we should enable this, It was enabled once but i thnk there were some issues

Reply:
Its safe to enable, you can test it on release